checker/match.go

```diff
@@ -175,7 +180,13 @@ func matchNamespace(nsMatch *namespaceMatch, ns namespace) bool {
 		"labels":    ns.Labels,
 		"rule":      nsMatch},
 	).Debug("Matching namespace.")
-	return matchName(nsMatch.Names, ns.Name) && matchLabels(nsMatch.Selector, ns.Labels)
+	//in case of plain text, Dikastes falls back on IP addresses. In such a case
```

Comment:
Minor point, but I think these comments don't match our typical style.

Namely a space after the `//`.

e.g., `// In case of plain text`